source/process/gitlab-process.rst

```diff
@@ -0,0 +1,116 @@
+
+Optionally, an engineer from both GitLab and Mattermost teams also joins the recurring monthly meeting.
+
+Service-Level Agreement (SLA)
+-------------------------------
+
+This process is still a work-in-progress, being developed together with GitLab.
+
+GitLab Premier Support Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+GitLab has a `4-hour support window for their premier support customers <https://about.gitlab.com/features/premium-support/>`_. Mattermost is setting up a process to meet those requirements for relevant questions raised by the GitLab premier support customers.
+
+GitLab is also working with their legal team to understand the process for sharing any sensitive customer information (e.g. logs) with Mattermost.
+
+More details to be added.
+
+GitLab Non-Premier Support Issues
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost has created a `mattermost-support` account in GitLab for support issues. The plan is for Mattermost to
+
+- integrate GitLab issue reviews as part of existing forum issue monitoring processes,
+- aim to answer within one business day, or at best effort, and
+- subscribe to email alerts sent automatically when a `Mattermost` label is added to an issue, or when the `mattermost-support` account is assigned to a GitLab issue.
+
+Templates
+--------------
+
+GitLab announcement proposal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: none
+
+  Proposed update for new version of [Mattermost](https://gitlab.com/gitlab-org/omnibus-gitlab/issues/1241).
+
+  ## GitLab Mattermost 2.2
+
+  [Mattermost 2.2](http://www.mattermost.org/mattermost-2-2-threaded-messages-and-more/) ships in GitLab 8.7 with threaded messages, French translation, new themes, new Trello and IRC support, plus many more new benefits.
+
+  This version also includes [security updates](http://about.mattermost.com/security-updates/) and upgrade from earlier versions is recommended.
+
+GitLab forum responses
+~~~~~~~~~~~~~~~~~~~~~~~
+
+See `sample forum responses listed here <https://docs.mattermost.com/process/community-guidelines.html#sample-responses>`_.
+
+Upcoming Work
+---------------
+
+1. Environment variables support for `config.json` settings. - **Due: Mattermost v4.10 / GitLab v11.0**
+
+ - `Disable Mattermost System Console settings that are configured via environment variables <https://mattermost.atlassian.net/browse/MM-9849>`_.
+ - `Investigate what config.json-only settings need to be added to the Mattermost System Console <https://mattermost.atlassian.net/browse/MM-9850>`_.
+ - `Support environment variable configuration options even if there's no entry for it in config.json <https://mattermost.atlassian.net/browse/MM-8400>`_.
+ - `Update documentation <https://gitlab.com/gitlab-org/omnibus-gitlab/issues/3284>`_ and fully test the migration.
+
+2. `Add timezones.json to GitLab Omnibus build <https://mattermost.atlassian.net/browse/MM-9873>`_ to support new Timezone user setting. - **Due: Mattermost v4.10 / GitLab v11.0**
```

Comment:
I'm actually going to close this ticket since it's not needed any more. `timezones.json` has been made optional, and a list of timezones has been hardcoded into the app